.gitignore

```diff
@@ -0,0 +1,2 @@
+nude_modules
```

Comment:
- [ ] This folder name should be `node_modules` and not `nude_modules`. Please correct this